shacl/DCAT-AP-NO-shacl_shapes_2.00.ttl

```diff
@@ -92,935 +99,1231 @@
         sh:severity sh:Violation
-    ], [
+    ], [ # sjekksumverdi
+        sh:datatype xsd:hexBinary ;
+        sh:maxCount 1 ;
         sh:minCount 1 ;
-        sh:class skos:Concept ;
-        sh:path dct:type ;
+        sh:path spdx:checksumValue ;
+        sh:message "spdx:checksumValue skal ha én og bare én verdi som skal være av type xsd:hexBinary"@nb ;
         sh:severity sh:Violation
     ] ;
-    sh:targetClass cpsv:Rule .
+    sh:targetClass spdx:Checksum .
 
-# Standard (dct:Standard), the whole class is an Norwegian extensions
+# Standard (dct:Standard)
 :Standard_Shape
     a sh:NodeShape ;
-    sh:name "Standard"@en , "Standard"@nb ;
-    sh:property [
+    rdfs:label "Standard"@en , "Standard"@nb ;
+    sh:property [ # har tittel
         sh:minCount 1;
         sh:nodeKind sh:Literal ;
         sh:path dct:title ;
+        sh:message "dct:title skal ha minst én verdi som skal være Literal"@nb ;
         sh:severity sh:Violation
-    ], [
+   ], [ # har referanse
+        sh:nodeKind sh:BlankNodeOrIRI;
+        sh:path rdfs:seeAlso ;
+        sh:message "rdfs:seeAlso skal være blank node eller IRI"@nb ;
+        sh:severity sh:Violation
+    ], [ # har versjonsnummer
         sh:maxCount 1 ;
         sh:nodeKind sh:Literal ;
         sh:path owl:versionInfo ;
+        sh:message "owl:versionInfo kan ha maks. én verdi som skal være Literal"@nb ;
+        sh:severity sh:Violation
+     ] ;
+    sh:targetObjectsOf dct:conformsTo ;
+ .
+
+# Tema (skos:Concept)
+# Ikke sjekk, fordi det skal velges fra et kontrollert vokab
+# :Category_Shape
+#     a sh:NodeShape ;
+#     rdfs:label "Theme"@en , "Tema"@nb ;
+#    sh:property [ # foretrukket tittel
+#        sh:minCount 1 ;
+#         sh:nodeKind sh:Literal ;
+#         sh:path skos:prefLabel ;
+#         sh:severity sh:Violation
+#     ] ;
+#    sh:targetClass skos:Concept .
+
+# Tematisk skjema (skos:ConceptScheme)
+# Ikke sjekk, fordi det skal velges fra et kontrollert vokab
+# :CategoryScheme_Shape
+#     a sh:NodeShape ;
+#     rdfs:label "Thematic Scheme"@en , "Tematisk skjema"@nb ;
+#     sh:property [ # tittel
+#         sh:minCount 1 ;
+#         sh:nodeKind sh:Literal ;
+#         sh:path dct:title ;
+#         sh:severity sh:Violation
+#     ] ;
+#     sh:targetClass skos:ConceptScheme .
+
+# Tidsrom (dct:PeriodOfTime)
+:PeriodOfTime_Shape
+    a sh:NodeShape ;
+    rdfs:label "Period of Time"@en , "Tidsrom"@nb ;
+    sh:property [ # sluttdato/tid, kardinalitet
+        sh:maxCount 1 ;
+        sh:path dcat:endDate ;
+        sh:severity sh:Violation ;
+        sh:message "dcat:endData kan ha maks. én verdi"@nb ;
+    ], [ # sluttdato/tid, datatype
+        sh:or ( [sh:datatype xsd:date]  [sh:datatype xsd:dateTime]  );        
+        sh:path dcat:endDate ;
+        sh:severity sh:Violation ;
+        sh:message "dcat:endDate skal være av type xsd:date eller xsd:dateTime"@nb ;
+    ] , [ # startdato/tid
```

Comment:
bør denne (1278 - 1282)  slås sammen med 1283 - 1287 slik at det blir likt som ellers i filen?
Alternativt innføre to sjekker overalt der det både er sjekk av maks/min og type/range.